core/src/main/java/nl/nn/adapterframework/doc/InfoBuilderSource.java

```diff
@@ -386,4 +397,25 @@ private static void getBeanProperties(Class<?> clazz, String verb, Map<String, M
 		}
 	}
 
+	public static boolean isGetterOrSetter(Method method) {
+		boolean isSetter = method.getReturnType().isPrimitive()
+				&& method.getReturnType().getName().equals("void")
+				&& (method.getParameterTypes().length == 1)
+				&& (method.getParameterTypes()[0].isPrimitive()
+						|| JAVA_BOXED.contains(method.getParameterTypes()[0].getName()));
+		boolean isGetter = (
+					(method.getReturnType().isPrimitive()
+							&& !method.getReturnType().getName().equals("void"))
+					|| JAVA_BOXED.contains(method.getReturnType().getName())
+				) && (method.getParameterTypes().length == 0);
+		return isSetter || isGetter;
+	}
+
+	public static boolean isConfigChildSetter(Method method) {
+		return (method.getParameterTypes().length == 1)
+				&& (! method.getParameterTypes()[0].isPrimitive())
+				&& (! JAVA_BOXED.contains(method.getParameterTypes()[0].getName()))
+				&& (method.getReturnType().isPrimitive())
+				&& (method.getReturnType().getName().equals("void"));
+	}
```

Comment:
Kunnen deze niet het beste ook naar Utils verhuizen?

Reply:
Op termijn moeten ze daar komen, maar nu is dat nog lastig. In de oude code ben ik `isGetterOrSetter` gaan gebruiken, om te zorgen dat de nieuwe @IbisDoc annotaties de Frank!Doc niet stuk maken. Die implementatie gebruikt de static variabele `JAVA_BOXED`. Alle methods die `JAVA_BOXED` ook gebruiken heb ik er hier maar bij gezet. In `Utils` zijn al methods die de methods uit `InfoBuilderSource` aanroepen. In de code die we hier ontwikkelen roep ik de methods uit `Utils` aan.